cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -482,18 +517,24 @@ def _configure_cluster(self):
         # create custom postgresql conf file with log settings
         fd, tmp_path = mkstemp()
         os.close(fd)
+        pg_params = {
```

Comment:
It might be good to move the generate out to here so that it's outside the with block as well - e.g.
pg_params = self._generate_pg_params({
  'log_destination'...
}

Reply:
Also, would it make sense for some more of these to go in the default params? POSTGRES_LOG_PATH is cluster-specific, but everything else can probably be in the defaults I think?

Reply:
@tehasdf could you please help me with that?  Should all of these parameters but `log_directory` be moved as @geokala suggests?

Reply:
sure why not